Pricing view

jonastore opened the pull request.

Commit:
first startpage buttons draft

Commit:
Improved button colors and background

jonastore committed:
add table to pricing-table view

jonastore committed:
add table to pricing-table view

jonastore committed:
add back-button

jonastore committed:
add back-button

jonastore committed:
removed buttons in favor of components

jonastore committed:
removed buttons in favor of components

jonastore committed:
centered text elements

jonastore committed:
centered text elements

jonastore committed:
add let's get started button component

jonastore committed:
add let's get started button component

jonastore committed:
add GetStarted button to Pricing view

jonastore committed:
add GetStarted button to Pricing view

jonastore committed:
add more navbar links

jonastore committed:
removed about us when logged in

jonastore committed:
add pricing link from downdown menu

jonastore committed:
changed register to sign up

jonastore committed:
removed some comment clutter

Commit:
started on a footer with icons and contact information

Commit:
modified package.json to enable yarn start outside of docker container

Commit:
Enable new route

	* import and export CreateQuestion view to index.js in Views/Site
	folder
	* add new route for and use CreateQuesiton as component

Commit:
change import order

Commit:
Create initial markup and layout for CreateQuestion view

	* add form and button components fro semantic-ui
	* add func to update state contected to inputs in form
	* add basic layout and styling

Commit:
add new scss file to style.scss and add style.css to .gitignore

Commit:
add func for adding more options

	* create function for adding more options when add options
	  button is clicked

Commit:
Merge branch 'dev' into feature/create-question-view-layout

	merge dev into feature/create-question-view-layout to solve
	problems with ressponsivnes

jonastore committed:
change order of links

jonastore committed:
commit to be able to merge

Commit:
fix layout for all devices

jonastore committed:
correct indentention

jonastore committed:
correct indentention

Commit:
Merge branch 'dev' into feature/startpage

Commit:
Merge branch 'dev' into feature/footer-layout

Commit:
remove unused container component

Commit:
reinstate PORT=7771 in start script

Commit:
add missing semicolon

Commit:
add uuid to package.json

Commit:
move func to smaler components

	* remove privius func
	* import question component

	this creates a cleaner view and helps to separete concerns

Commit:
Refactor createquestion func

	* move func for updating option value to CreateOption component
	* rework how new options are added to the form
	* add func for removing options

	This commit improves the overview of the functionality by spliting the code into different
	components and adds func for removing an option

Commit:
improve responsivenes for createQuestion view

Commit:
bugfix question input value unable to update

	* add func for updating state for inputs

	this func was preaviusly removed by mistake, now it's added back in

izabelrosen committed:
Merge pull request #4 from chas-academy/feature/create-question-view-layout

Feature/create question view layout

Commit:
fixed issue with responsive field

Commit:
made the buttons responsive

Commit:
add eslintrc

SebastianGerS commented on package.json:
```diff
@@ -29,6 +30,12 @@
     "redux-thunk": "^2.2.0",
+    "semantic-ui-css": "^2.3.1",
+    "semantic-ui-react": "^0.79.1",
+<<<<<<< HEAD
```
you have to resolve the merge conflict here

SebastianGerS commented on src/Components/Buttons/GetStarted.js:
```diff
@@ -0,0 +1,18 @@
+const GetStarted = () => (
+  <div className='ui center aligned basic segment'>
+
+      <Button padded centered size='huge' className='centered'>
```
padded  and centered  on the button component gives a warning when compiled.
I don't think those values can be associated with the semantic button buttons are always padded if the  compact attribute is not set.

SebastianGerS commented on src/Components/Buttons/GetStarted.js:
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { Header, Table, Icon, Button } from 'semantic-ui-react'
```
Header, table, icon are defined butt never used, please remove

SebastianGerS commented on src/Views/Site/Pricing.js:
```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import GetStarted from '../../Components/Buttons/GetStarted'
+
+import { Header, Table, Icon, Button } from 'semantic-ui-react'
```
Button is defined but never used, please remove.

SebastianGerS reviewed (commented):
Looks good, the responsiveness seams to work now, just some minor things, fix and then push, no need to do a new pull request

jonastore committed:
solve merge issues

jonastore committed:
solve merge issues

jonastore committed:
remove defined but unused stuff

Commit:
configure eslint,flow and prettier

	* update .eslintrc
	* add .babelrc
	* add plugins and config for eslint via yarn
	* add eslint, flow with yarn
	* move bootstrap to to regular dependencies (no-extreanous-dependencies)

Commit:
adjust files to follow airbnb coding standard

	* adjust all files to comply with code standard (in somcases comments to ignore rules where added)

	this was done to comply with the airbnb coding standard, the ignored rules will be corrected for in time

Commit:
initialize flow

Commit:
add precommit hook

	* add husky and lint-staged
	* add precomit script
	* add config for lint-staged

jonastore committed:
add styles to gitignore

jonastore committed:
add styles to gitignore

Commit:
add prepush-hook

Commit:
remove non functioning prepushook

Commit:
add workspace settings

Commit:
update reademe with information about workspace settings, linting etc

Commit:
add rules for prettier and test precommit-hook

Commit:
test new eslint settings

Commit:
test new eslint settings

jonastore committed:
resolve more merge conflicts

jonastore committed:
Merge branch 'dev' into pricing-view

jonastore committed:
Merge branch 'dev' into feature/header-component

izabelrosen committed:
Merge branch 'dev' of github.com:chas-academy/platinum-client into dev

izabelrosen committed:
Update exoframe variable names

jonastore committed:
Merge branch 'feature/footer-layout' of github.com:chas-academy/platinum-client into pricing-view

izabelrosen committed:
Remove .env mapping from docker-compose file

Commit:
fixed issues with landscape mode, and centered layout with smaller screens

jonastore committed:
Merge branch 'feature/footer-layout' of github.com:chas-academy/platinum-client into pricing-view

Commit:
less line-height and height

Commit:
fixed indenting

izabelrosen committed:
Remove ports from docker-compose

Commit:
fixed indenting, removed background image for now

izabelrosen committed:
Add ports

izabelrosen committed:
Remove ports

izabelrosen committed:
add ports

izabelrosen committed:
Problem with ports

izabelrosen committed:
add port to docker_compose and remove port from start script

izabelrosen committed:
change port in docker-compose

izabelrosen committed:
Remove port in docker-compose

Commit:
replace pretty formating with eslint fix for precommit formating

Commit:
testing precommit formating

Commit:
updated reame with instructions regarding eslint,prettier,flow and husky

Commit:
add more instructions to readme...

Commit:
add background image without use of !important

	* add new styling rule to App.scss to add in a background image

Commit:
Merge branch 'feature/footer-layout' into dev

	This merge incorporates the new styling for the footer

Commit:
Merge branch 'feature/header-component' into dev

	This merge adds the new styling of the header to the dev branch

Commit:
remove invalid centered prop from get started button

SebastianGerS reviewed (approved):
look good!

made a smal commit to adjust the getstarted button, everything else looks great!

jonastore committed:
remove invalid centered prop from get started button

jonastore committed:
remove invalid centered prop from get started button

jonastore committed:
fix merge conflict

izabelrosen commented:
Good work - but please make the table more centered of the page!

Commit:
Merge branch 'config/enforce-coding-standards' into dev

	resolve merge-conflict due to new linting settings aplyed in new files

jonastore committed:
try fixing vertical alignment of pricing table

jonastore committed:
work on centering tabe

jonastore committed:
fix merge issues

jonastore committed:
fix merge issues

jonastore committed:
change page headline

jonastore committed:
add proper headline margins

jonastore committed:
change headline

jonastore committed:
separate layout and components
